libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -282,32 +292,26 @@ class MapboxTripNotification constructor(
     }
 
     private fun getManeuverResource(step: LegStep): Int {
-        val maneuver = step.maneuver()
-        val maneuverType = maneuver.type()
-        maneuver.let { stepManeuver ->
-            val maneuverModifier = stepManeuver.modifier()
-            if (!TextUtils.isEmpty(maneuverModifier)) {
-                val drivingSide = step.drivingSide()
-                val isLeftSideDriving = isLeftDrivingSideAndRoundaboutOrRotaryOrUturn(
-                    maneuverType,
-                    maneuverModifier,
-                    drivingSide
-                )
-                return when (isLeftSideDriving) {
-                    true -> {
-                        ManeuverResource.obtainManeuverResource(
-                            maneuverType + maneuverModifier + drivingSide
-                        )
-                    }
-                    else -> {
-                        ManeuverResource.obtainManeuverResource(
-                            maneuverType + maneuverModifier
-                        )
-                    }
-                }
+        val stepManeuver = step.maneuver()
+        val maneuverType = stepManeuver.type()
+        val maneuverModifier = stepManeuver.modifier()
```

Comment:
What happens if stepManeuver is null? Is it worth doing the following in that case?

Reply:
It is marked as NonNull in LegStep.java. Let's trust this annotation